app/utils/emoji_utils.js

```diff
@@ -46,7 +46,7 @@ function isEmoticon(text) {
 }
 
 export function hasEmojisOnly(message, customEmojis) {
-    if (!message || message.length === 0) {
+    if (!message || message.length === 0 || /^\s{4}/.test(message)) {
```

Comment:
Looks good, but ESLint is reporting an error here
```suggestion
    if (!message || message.length === 0 || (/^\s{4}/).test(message)) {
```